.github/workflows/main/installArnold.py

```diff
@@ -68,6 +69,11 @@
 
 print( "Downloading Arnold \"{}\"".format( url ) )
 
+context = ssl.create_default_context()
```

Comment:
Seems we might have a rogue commit here?

Reply:
Ah yes, that was left over from when I needed to push it to an Arnold branch for user testing. Removed in the latest push along with a couple others that were superfluous (fixing `.gfr` whitespace and a cherry pick from #5997).